packages/frontend-2/components/connectors/Page.vue

```diff
@@ -38,6 +38,7 @@
             :items="categories"
             size="base"
             color="foundation"
+            clearable
```

Comment:
Small papercut, because it's min width it jumps when the clear button is shown, you can change it to a fixed width instead of max